### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by ch4ot1c**

### CODE REVIEW

File: contrib/build-osx/make_osx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+cd $build_dir/../..
+
+export PYTHONHASHSEED=22
+VERSION=`git describe --tags`
+
+# Paramterize
+PYTHON_VERSION=3.6.4
+BUILDDIR=/tmp/electrum-build
+PACKAGE=Electrum
+GIT_REPO=https://github.com/spesmilo/electrum
```
Comment (KaboomFox) [inline contrib/build-osx/make_osx]: we will need to change this.

File: contrib/build-osx/make_osx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+rm -rf ./dist
+
+
+rm  -rf $BUILDDIR > /dev/null 2>&1
+mkdir $BUILDDIR
+
+info "Downloading icons and locale..."
+for repo in icons locale; do
+  git clone $GIT_REPO-$repo $BUILDDIR/electrum-$repo
```
Comment (KaboomFox) [inline contrib/build-osx/make_osx:56]: here too

Comment (nimbosa) [reply contrib/build-osx/make_osx:56]: need to confirm if changes here fixes this: https://github.com/z-classic/electrum-zcl/pull/153/commits/962b8eaa306dfa61fd3feba55f7aae5eae48b820

```suggestion
  git clone $GIT_REPO-$repo $BUILDDIR/electrum-$repo
```
where is `$repo` coming from?

**Comment by nimbosa** [comment]
single followup commits may go directly to `develop` branch, _feature-_ or _fix-_ branches may be necessary for larger changes over multiple commits

**Comment by nimbosa** [comment]
we are building on top of upstream Electrum 3.1.1 now

**Pull request merged**